### PR TITLE
test: migrate from puppeteer to playwright

### PR DESCRIPTION
**Pull request by LiKang6688**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -68,13 +68,18 @@ jobs:
             fi
           environment:
             JEST_JUNIT_OUTPUT_DIR: ./coverage/junit/
+      - run:
+          name: Install playwright
+          command: npx playwright install
```
Comment (niekvanstaveren) [inline .circleci/config.yml:73]: Why are we using npx to install this? Can we have it through npm dependencies instead?

File: .github/CONTRIBUTION.md [hunk 2]
```diff
@@ -8,15 +8,24 @@
...
 
-`yarn test:rendering`
+    # Install dependencies
+    yarn --frozen-lockfile
+
+    # Build nebula.js visualization
+    yarn build
+
+    chmod 777 ./test/rendering/commands/run-rendering-test.sh
+    yarn test:local:rendering
```
Comment (niekvanstaveren) [inline .github/CONTRIBUTION.md:26]: Should this be one copy block?

File: .circleci/config.yml (hunk 1 above)
Comment (LiKang6688) [reply .circleci/config.yml:73]: the name is misleading which is from their docs. 
it is actually installing supported browsers.
I will fix it to install supported browsers.
https://playwright.dev/docs/ci

File: .github/CONTRIBUTION.md (hunk 2 above)
Comment (LiKang6688) [reply .github/CONTRIBUTION.md:26]: Screenshot 2022-05-03 at 11 50 08
it is one copy block. Do I understand you correctly?

**Review by niekvanstaveren — APPROVED** [review]

**Pull request merged**